test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/sso/s2i/KieServerS2iWithSsoDroolsIntegrationTest.java

```diff
@@ -108,27 +112,15 @@
     private KieCommands commandsFactory = KieServices.Factory.get().getCommands();
     private Set<Class<?>> extraClasses;
 
-    private static final String SECURED_URL_PREFIX = "secured-";
-    private static final String RANDOM_URL_PREFIX = UUID.randomUUID().toString().substring(0, 4) + "-";
-    private static final String KIE_SERVER_NAME = "kieserver-sso-drools";
-    private static final String KIE_SERVER_HOSTNAME = KIE_SERVER_NAME + DeploymentConstants.getDefaultDomainSuffix();
-
     @Override
-    protected GenericScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+    protected ImmutableKieServerScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
         repositoryName = Git.getProvider().createGitRepositoryWithPrefix("KieServerS2iDroolsRepository", KieServerS2iWithSsoDroolsIntegrationTest.class.getResource(PROJECT_SOURCE_FOLDER).getFile());
 
-        DeploymentSettings kieServerS2Isettings = kieServerS2ISettingsBuilder
-                .withContainerDeployment(KIE_CONTAINER_DEPLOYMENT)
-                .withSourceLocation(Git.getProvider().getRepositoryUrl(repositoryName), REPO_BRANCH, DEPLOYED_KJAR.getName())
-                .withDroolsServerFilterClasses(false)
-                .withHostame(RANDOM_URL_PREFIX + KIE_SERVER_HOSTNAME)
-                .withSecuredHostame(SECURED_URL_PREFIX + RANDOM_URL_PREFIX + KIE_SERVER_HOSTNAME)
```

Comment:
We should leave in this test configuration of custom hostnames to check that custom hostnames are correctly set for Deployment and in SSO

Reply:
Custom hostnames returned back.